doc/ref_model/chapters/chapter02.md

```diff
@@ -266,17 +266,44 @@ By trying to sort workloads into different categories based on the requirements
 <a name="2.4"></a>
 ## 2.4 Cloud Infrastructure Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads.
+Two profile layers are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, ie: an infrastructure object can belong to one and only one profile, and workloads can request to be instantiated on one infrastructure profile only.
```

Comment:
```suggestion
- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, ie: an infrastructure object can belong to one and only one profile, where VMs can then be instantiated on one infrastructure profile only.
```

Reply:
why only VMs and not e.g. pods?

Reply:
@rgstori yes should be both VMs and Pods